docs/services/element.md

```diff
@@ -0,0 +1,33 @@
+## Element WebClient 
+```yaml 
+version: "3.9"
+
+services:
+  element:
+    container_name: element
+    image: vectorim/element-web
+    restart: always
+    volumes:
+      - '/srv/comms/matrix/element/data/config.json:/app/conig.json'
```

Comment:
Change `/srv/comms/matrix/element/` to `/srv/comms/element/`, because this is a service that has basicly nothing to do with the matrix service.